src/server/graphql/models/Team/teamSchema.js

```diff
@@ -46,3 +47,22 @@ const teamInputThunk = () => ({
 
 export const CreateTeamInput = nonnullifyInputThunk('CreateTeamInput', teamInputThunk, ['id', 'name']);
 export const UpdateTeamInput = nonnullifyInputThunk('UpdateTeamInput', teamInputThunk, ['id']);
+
+export const CreateTeamOutput = new GraphQLObjectType({
```

Comment:
I'm not sure I love this. This output is a subset of an entity, but it's not an entity itself (ie we wouldn't ask "hey `user`, who's the leader of your `createTeamOutput`?). If we need a subset of a userprofile, maybe we could write a query string for the `WelcomeContainer` that is a subset of the `getAuthedUser` query string. I'll PR something up